rasa/shared/core/slots.py

```diff
@@ -1,10 +1,10 @@
 import logging
-from typing import Any, Dict, List, NoReturn, Optional, Text, Type
 
-from rasa.core import utils
 from typing import Any, Dict, List, NoReturn, Optional, Text, Type
-from rasa.core.constants import DEFAULT_CATEGORICAL_SLOT_VALUE
-from rasa.utils.common import class_from_module_path, raise_warning
+
+import rasa.shared.utils.common
+from rasa.shared.core.constants import DEFAULT_CATEGORICAL_SLOT_VALUE
```

Comment:
again, I find it better to import the constants module itself, not separate constants